docs/reference/modules/snapshots.asciidoc

```diff
@@ -323,33 +323,40 @@ POST /_snapshot/my_backup/snapshot_1/_restore
 -----------------------------------
 // CONSOLE
 
-By default, all indices in the snapshot as well as cluster state are restored. It's possible to select indices that
-should be restored as well as prevent global cluster state from being restored by using `indices` and
-`include_global_state` options in the restore request body. The list of indices supports
-<<search-multi-index-type,multi index syntax>>. The `rename_pattern` and `rename_replacement` options can be also used to
-rename index on restore using regular expression that supports referencing the original text as explained
+By default, all indices in the snapshot are restored, the cluster state is not
+restored. It's possible to select indices that should be restored as well as
+prevent global cluster state from being restored by using `indices` and
```

Comment:
this should read: It's possible .... as well as **allow** global cluster state to be restored ...